### ISSUE TITLE
Missing balance after upgrading to wallet 1.0.0.10

### ISSUE THREAD

**Issue by n1njam1rko** [Veil-Project/veil#295]
I'm missing 19.8911513 Veil after updating to latest version of the wallet (1.0.0.10). Explorer shows that balance for my base address, but wallet does not.

As suggested by support tried all the commands but without success.

In wallet shows "received with anonymous" for that missing amount and shows enough confirmations.
![anonim](https://user-images.githubusercontent.com/46599210/51202460-133cf580-18ff-11e9-8891-3946a8ff6739.PNG)
In explorer shows: "unspent" for that transaction.
![unspent](https://user-images.githubusercontent.com/46599210/51202522-3667a500-18ff-11e9-9397-4f75857959f7.PNG)
from wallet, after rescan and running command "listaddressgroupings"
![rescan](https://user-images.githubusercontent.com/46599210/51202620-7595f600-18ff-11e9-9373-0a3a39eba9a0.PNG)

1.0.0.10, release from github.

Windows 10, Ubuntu 16

Prior to mining 20 zerocoins, I received 2 payments from pools, 19.8911513 from suprnova and 1.06 from veilmine, then I minted 20 zero coins. In the morning, after upgrading wallet, I see that I didn't get zero coins and I have missing basecoins. Maybe zerocoins are not minted because smaller of those 2 transactions wasn't confirmed, and tx for it doesn't exist.

**Comment by seanPhill** [comment]
Can you try ```listzerocoinamounts```? If they don't show up as "pending" or as "spendable" then try the following. I found my missing testnet zerocoin(s) when I did "rescanzerocoinwallet". (The 5000 number may be changed to just 20 or 30 to search for just a small number of zerocoins.)
```
rescanblockchain
getarchivedzerocoin
rescanzerocoinwallet
searchdeterministiczerocoin 0 5000 2
```? Your missing basecoin has been subtracted for the zerocoin. Now to

**Comment by n1njam1rko** [comment]
Tried all those commands, and few others, many times, but no difference.
Now tried only those, nothing changed.
Yes, basecoin has been subtracted when I made them into zerocoin, but that zero coin don't have any confirmations, that txid don't exist on explorer.
I suspect that it has to do something with the fact that at the time I minted them wallet had enough balance, but later one of the basecoin transactions got invalidated (the one bellow green one on the picture) so zerocoin also became invalid.
But, shouldn't those basecoins got returned back or something like that?
Also it's strange that those 2 transactions are _Basecoin received with anonymous_ although I used the same address as usual for payments from mining pools.
